concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -253,6 +244,48 @@ def test_that_concent_reponds_with_too_small_requestor_deposit_when_requestor_do
         )
         self._assert_stored_message_counter_not_increased()
 
+    def test_that_concent_responds_with_too_small_provider_deposit_when_provider_does_not_have_funds(self):
+        """
+        Provider -> Concent: SubtaskResultsVerify
+        Concent -> Provider: ServiceRefused (TooSmallProviderDeposit)
+        """
+        # given
+        (serialized_subtask_results_verify,
+         subtask_results_verify_time_str) = self._create_serialized_subtask_results_verify()
+
+        # when
+        with mock.patch(
+            "core.message_handlers.bankster.claim_deposit",
+            return_value=[True, False]
```

Comment:
I don't mean to be too picky, but next time it would be better to have `return_value` of the same type as the function being mocked (`tuple` vs `list`). Don't bother to change it though ;)

Reply:
I agree, good catch :+1: done